File: bee1004.java
// 1004 - Produto Simples
// https://www.beecrowd.com.br/judge/pt/problems/view/1004

import java.util.Scanner;

public class bee1004 {

    public static void main(String[] args) {

        int A, B, PROD = 0;

        try (Scanner entrada = new Scanner(System.in)) {
            A = entrada.nextInt();
            B = entrada.nextInt();
        }

        PROD = A * B;

        System.out.printf("PROD = " + PROD + "\n");
    }
}

File: bee1004.cs
// 1004 - Produto Simples
// https://www.beecrowd.com.br/judge/pt/problems/view/1004

using System;

class bee1004
{

    static void Main(string[] args)
    {
        int A = Convert.ToInt32(Console.ReadLine());
        int B = Convert.ToInt32(Console.ReadLine());

        int PROD = A * B;

        Console.WriteLine("PROD = " + PROD);
    }
}